utils/test_on_server.js

```diff
@@ -292,18 +306,25 @@ async function connectToCli() {
   socket.on('data', async (data) => {
     data = data.toString('utf8');
     const line = data.replace(/^< /, '').replace(/\n< /, '');
-    if (spawnBots(line, socket)) {
+    if (await spawnBots(line, socket)) {
       return;
     }
     if (setPassword(line, socket)) {
```

Comment:
why not await here?
`await setPassword...` and `const setPassword = async function(line, socket) { ` on https://github.com/TooAngel/screeps/pull/510/files#diff-b532caf1840c952b1bfc47fa31a0119fR231

Reply:
I need to change the `spawnBots` to an `async` function, to be able to use `await`. This is not necessary for `setPassword` (yet).